initial draft for Privileged Identity Management (PIM)

mingtm opened the pull request:
This is the initial draft for Privileged Identity Management APIs that are targeted to publish with MSGraph beta.

msftclas commented:
Hi __@mingtm__, I'm your friendly neighborhood Microsoft Pull Request Bot (You can call me MSBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.microsoft.com.
    </span>

TTYL, MSBOT;

msftclas commented:
__@mingtm__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, MSBOT;

mingtm commented:
From: Ryan Gregg 
Sent: Monday, May 23, 2016 1:29 PM
To: Sudhi Ramamurthy <suramam@exchange.microsoft.com>
Cc: Ming Tang <mingt@exchange.microsoft.com>; Qi Cao <qicao@microsoft.com>
Subject: RE: Docs for MS Graph preview

Oops, sorry about that. I updated the tooling to be versioned now, but missed updating the Graph scripts to support the new version numbers. I've updated the scripts and the builds should run normally now. You can rerun any existing builds that failed to make sure they are clean before merging pull requests.

-Ryan

msftclas commented:
Hi __@mingtm__, I'm your friendly neighborhood Microsoft Pull Request Bot (You can call me MSBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, MSBOT;

mingtm commented:
need to fix link issues

msftclas commented:
Hi __@mingtm__, I'm your friendly neighborhood Microsoft Pull Request Bot (You can call me MSBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, MSBOT;

mingtm commented:
close due to tool test issues.

msftclas commented:
Hi __@mingtm__, I'm your friendly neighborhood Microsoft Pull Request Bot (You can call me MSBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, MSBOT;

mingtm commented:
need to workaround tool bugs

msftclas commented:
Hi __@mingtm__, I'm your friendly neighborhood Microsoft Pull Request Bot (You can call me MSBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, MSBOT;
